packages/squiggle-lang/src/fr/tag.ts

```diff
@@ -133,6 +139,36 @@ export const library = [
       }),
     ],
   }),
+  maker.make({
+    name: "format",
+    examples: [],
+    definitions: [
+      makeDefinition(
+        [frForceBoxed(frDistOrNumber), frNamed("numberFormat", frString)],
+        frForceBoxed(frDistOrNumber),
+        ([{ args, value }, format]) => {
+          checkNumericTickFormat(format);
+          return { args: args.merge({ numberFormat: format }), value };
+        }
+      ),
+      makeDefinition(
+        [frForceBoxed(frDuration), frNamed("numberFormat", frString)],
```

Comment:
We could have merged this with  the above, something like, ``frOr(frNumber, frDist, frDuration)``, but that didn't feel right. I imagine it's likely we'll change this later.